synthesizer/src/process/tests.rs

```diff
@@ -225,14 +229,14 @@ fn test_process_multirecords() {
     assert_eq!(authorization.len(), 1);
 
     // Execute the request.
-    let (response, execution, _inclusion, _metrics) = process.execute::<CurrentAleo, _>(authorization, rng).unwrap();
+    let (response, _trace, _metrics) = process.execute::<CurrentAleo>(authorization).unwrap();
     let candidate = response.outputs();
     assert_eq!(3, candidate.len());
     assert_eq!(output_a, candidate[0]);
     assert_eq!(output_b, candidate[1]);
     assert_eq!(output_c, candidate[2]);
 
-    process.verify_execution::<false>(&execution).unwrap();
+    // process.verify_execution::<false>(&execution).unwrap();
```

Comment:
These should all be uncommented again

Reply:
These can actually be removed. We weren't checking the proofs prior, hence the `false`. We merely verified that it was possible to construct the verifier inputs without fail. (there's not enough structure here to check a real proof)